Final status:
Process still alive after running test suite
stdout and stderr:
DEBUG:root:[*] running method core_negotiate_tlv_encryption
DEBUG:root:[*] Negotiating TLV encryption
DEBUG:root:[*] RSA key: 30820122300d06092a864886f70d01010105000382010f003082010a0282010100a641ce741914c69ca8c814016811c3d4c028b8ff90246eadccd4c2f581bb5826736b7dc3679b722d27c9d8e11eaa4189797d3b2b86fe53ac41c4a93202af64762f6cb5547252857a54235b9792704a3dda0cc2e0ec4e190e4aad9e3d43ad35a1f65bde0dfb5ab7840e5cb7742facde12fd40ee77b7b356efc41ef6b43285429ab3ec45fab326bf7efb10aaccf4c9b9c9a412b4dc08a3629d6b8514cee49f8d01f7b0ba9bebe554ae8d9cdd6f2388865e37ae420035ac3af54d04de75004df36805b3c5c7e1cddbb68ffc22e721dd2e183aa83bd026ba6ae75de2d7f518da3c72eed33525c1477884c9393b68ea918044f6bba46cf64170b677fd2a4dacc4ae590203010001
DEBUG:root:[*] AES key: 0x405da8064fe1493af40d78f4b01c9e17540983b278d1d86e483bcbf8f9c8af4e
DEBUG:root:[*] Encrypted AES key: 0x9ae68dc39dc6559542b0e086c5562de21ebdc1cb7d95e7ce9c3c6fcc565b2c4172ac615a81ff0e326ed697466d963705f4d12986bcfd52dc69308a1919eb669faeb119d9d5678d057e6d3ee62ac6be87af41e666d6f03170ebda929094dc80cd1845b8783f8ff720358aaa022de66077266ed627e5efc6c864a677960e5420e1b8919153bae2ed22c65f31daf37569a682c72872a55f719c94309b1b60bbc723caa17cc186bd2bf11dbc2b33606760dc6162c18fb967aad71404e514d48aa6fda20a4312653966229345cec46497bd29446dafcf40fd63134ed512896c624a107509b0c9d2c63c50ffcb1890d8bd9d243f79c87a41878de48d604897a4e14f82
DEBUG:root:[*] TLV encryption sorted
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_set_session_guid
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_enumextcmd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_enumextcmd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_loadlib
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getuid
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_sysinfo
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_set_uuid
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_net_config_get_interfaces
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_net_config_get_routes
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_separator
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_chdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir-esibkugd'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_chdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 1 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 2 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 2
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 3 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 3
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 4 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 4
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 5 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 5
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\boot.ini'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\pagefile.sys'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '/etc/master.passwd'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\notepad.exe'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\calc.exe'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 6 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 6
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 7 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 7
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 8 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 8
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 9 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 9
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 10 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 10
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 11 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 11
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 12 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 12
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_file_move
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 13 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 13
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir/directory'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 14 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 14
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_ls
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_execute
DEBUG:root:[*] starting process: ['/bin/sh', '-c', 'command -v ln || which ln && echo FKPYHYYZ']
DEBUG:root:[*] added process id: 3437, handle: 1
DEBUG:root:[*] added channel id: 15 type: MeterpreterProcess
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[-] method core_channel_read resulted in error: #1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_close
DEBUG:root:[*] closed and removed process handle: 1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 16 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 16
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_execute
DEBUG:root:[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir $(pwd)/meterpreter-test-dir.lnk']
DEBUG:root:[*] added process id: 3440, handle: 2
DEBUG:root:[*] added channel id: 17 type: MeterpreterProcess
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_close
DEBUG:root:[*] closed and removed process handle: 2
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.lnk'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.1'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.2'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 18 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 18
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_execute
DEBUG:root:[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir.1 $(pwd)/meterpreter-test-dir.2/link']
DEBUG:root:[*] added process id: 3446, handle: 3
DEBUG:root:[*] added channel id: 19 type: MeterpreterProcess
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_close
DEBUG:root:[*] closed and removed process handle: 3
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_chdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
